src/modules/fw_att_control/FixedwingAttitudeControl.cpp

```diff
@@ -466,30 +468,31 @@ FixedwingAttitudeControl::vehicle_land_detected_poll()
 	}
 }
 
-void FixedwingAttitudeControl::get_airspeed_and_scaling(float &airspeed, float &airspeed_scaling)
+float FixedwingAttitudeControl::get_airspeed_and_update_scaling()
 {
+	_airspeed_sub.update();
 	const bool airspeed_valid = PX4_ISFINITE(_airspeed_sub.get().indicated_airspeed_m_s)
-				    && (_airspeed_sub.get().indicated_airspeed_m_s > 0.0f)
-				    && (hrt_elapsed_time(&_airspeed_sub.get().timestamp) < 1e6);
+				    && (_airspeed_sub.get().indicated_airspeed_m_s >= 0.0f)
```

Comment:
TODO: if landed tolerate a small negative airspeed to prevent immediate invalidation?

@Antiheavy thoughts?

Reply:
No, I would take it out completely.